Consul/Lock.cs

```diff
@@ -251,23 +252,24 @@ public async Task<CancellationToken> Acquire(CancellationToken ct)
 
                     var qOpts = new QueryOptions()
                     {
-                        WaitTime = Opts.LockWaitTime
+                        WaitTime = Opts.LockWaitTime,
```

Comment:
Please remove the comma at the end for style consistency with the rest of the project

Reply:
👍